handler.py

```diff
@@ -267,6 +265,22 @@ def run_inference(loader, model_wrapper, write_output=False, mode='loc', return_
     else:
         return_dict[f'{model_wrapper.model_size}{mode}'] = results_list
 
+
+def check_data(images):
+    """
+    Check that our image pairs contain useful data. Note: This only check the first band of each file.
+    :param images: Images to check for data
+    :return: True if both images contain useful data. False if either contains no useful date.
+    """
+    for image in images:
+        with rasterio.open(image) as src:
```

Comment:
I think this would be better in the actual model code. We'll really slow down the inference if we do I/O twice. Can you not do this check on the part of the code that does the model inference?

Reply:
My though process was that if either image contains no data we don't even send that pair to the inference. With my limited knowledge as far as Jared's code is concerned, it looks to me as if we still run the inference, just on two images that contain no useful data. Am I reading that correct?

Reply:
You're right. What it does is: if the post is black, it makes the pre black too. That way localization won't find any buildings. That's essentially "not running" prediction, though idk if it's faster.